docs/src/content/docs/de/reference/frontmatter.md

```diff
@@ -0,0 +1,301 @@
+
+```md
+---
+# Versteckt den Link zur vorherigen Seite
+prev: false
+---
+```
+
+```md
+---
+# Überschreibe den Linktext der vorherigen Seite
+prev: Fortsetzung des Tutorials
+---
+```
+
+```md
+---
+# Überschreibe sowohl den Link zur vorherigen Seite als auch den Text
+prev:
+  link: /unverwandte-seite/
+  label: Schau dir diese andere Seite an
+---
+```
+
+### `next`
+
+**type:** `boolean | string | { link?: string; label?: string }`
+
+Dasselbe wie [`prev`](#prev), aber für den Link zur nächsten Seite.
+
+```md
+---
+# Versteckt den Link zur nächsten Seite
+next: false
+---
+```
+
+### `pagefind`
+
+**type:** `boolean`  
+**default:** `true`
+
+Legt fest, ob diese Seite in den [Pagefind](https://pagefind.app/)-Suchindex aufgenommen werden soll. Setze das Feld auf `false`, um eine Seite von den Suchergebnissen auszuschließen:
+
+```md
+---
+# Diese Seite aus dem Suchindex ausblenden
+pagefind: false
+---
+```
+
+### `sidebar`
+
+**type:** `{ label?: string; order?: number; hidden?: boolean; badge?: string | BadgeConfig }`
+
+Steuert, wie diese Seite in der [Seitenleiste](/de/reference/configuration/#sidebar) angezeigt wird, wenn eine automatisch generierte Linkgruppe verwendet wird.
+
+#### `label`
+
+**type:** `string`  
+**default:** the page [`title`](#title-erforderlich)
+
+Legt die Bezeichnung für diese Seite in der Seitenleiste fest, wenn sie in einer automatisch erzeugten Linkgruppe angezeigt wird.
+
+```md
+---
+title: Über dieses Projekt
+sidebar:
+  label: Infos
+---
+```
+
+#### `order`
+
+**type:** `number`
+
+Steuere die Reihenfolge dieser Seite beim Sortieren einer automatisch erstellten Gruppe von Links.
+Niedrigere Nummern werden in der Linkgruppe weiter oben angezeigt.
+
+```md
+---
+title: Erste Seite
+sidebar:
+  order: 1
+---
+```
+
+#### `hidden`
+
+**type:** `boolean`  
+**default:** `false`
+
+Verhindert, dass diese Seite in eine automatisch generierte Seitenleistengruppe aufgenommen wird.
+
+```md
+---
+title: Versteckte Seite
+sidebar:
+  hidden: true
+---
+```
+
+#### `badge`
+
+**type:** <code>string | <a href="/de/reference/configuration/#badgeconfig">BadgeConfig</a></code>
+
+Füge der Seite in der Seitenleiste ein Abzeichen hinzu, wenn es in einer automatisch generierten Gruppe von Links angezeigt wird.
+Bei Verwendung einer Zeichenkette wird das Abzeichen mit einer Standard-Akzentfarbe angezeigt.
+Optional kann ein [`BadgeConfig` Objekt](/de/reference/configuration/#badgeconfig) mit den Feldern `text` und `variant` übergeben werden, um das Abzeichen anzupassen.
+
+```md
+---
+title: Seite mit einem Badge
+sidebar:
+  # Verwendet die Standardvariante, die der Akzentfarbe deiner Website entspricht
+  badge: Neu
+---
+```
+
+```md
+---
+title: Seite mit einem Abzeichen
+sidebar:
+  badge:
+    text: Experimentell
+    variant: caution
+---
+```
```

Comment:
Hier fehlt eine kurze [neue Abschnitt](https://starlight.astro.build/reference/frontmatter/#attrs):

````md
#### `attrs`

**type:** `Record<string, string | number | boolean | undefined>`

HTML attributes to add to the page link in the sidebar when displayed in an autogenerated group of links.

```md
---
title: Page opening in a new tab
sidebar:
  # Opens the page in a new tab
  attrs:
    target: _blank
---
```
````